arrow-avro/src/reader/vlq.rs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+/// Decoder for zig-zag encoded VLQ integers
+///
+/// <https://protobuf.dev/programming-guides/encoding/#varints>
```

Comment:
FWIW the avro docs include a reference to lucene (not protobuf): https://lucene.apache.org/core/3_5_0/fileformats.html#VInt

Maybe not a big deal, but it was weird to see the protobuf reference here

```suggestion
/// Decoder for zig-zag encoded variable length (VLW) integers
/// See also: 
/// <https://avro.apache.org/docs/1.11.1/specification/#primitive-types-1>
/// <https://protobuf.dev/programming-guides/encoding/#varints>
```

Reply:
> Maybe not a big deal, but it was weird to see the protobuf reference here

Yeah, amusingly the Avro spec links to protobuf as well. Avro is very heavily inspired by protobuf, and isn't ashamed to admit it :smile: